python/ray/tune/tests/test_tune_save_restore.py

```diff
@@ -50,57 +53,50 @@ def tearDown(self):
         # Without this line, test_tune_server.testAddTrial would fail.
         _register_all()
 
-    def _get_trial_dir(self, absoulte_exp_dir):
-        trial_dirname = next(
-            (child_dir for child_dir in os.listdir(absoulte_exp_dir)
-             if (os.path.isdir(os.path.join(absoulte_exp_dir, child_dir))
-                 and child_dir.startswith(self.MockTrainable._name))))
-
-        trial_absolute_dir = os.path.join(absoulte_exp_dir, trial_dirname)
-
-        return trial_dirname, trial_absolute_dir
+    def _get_trial_dir(self, absolute_exp_dir):
+        dirs = glob(absolute_exp_dir + "/" + self.MockTrainable._name + "*")
+        absolute_trial_dir = dirs[0]
+        return absolute_trial_dir
 
-    def _train(self, exp_name, local_dir, absolute_local_dir):
+    def _train(self, exp_name, absolute_local_dir):
         trial, = tune.run(
             self.MockTrainable,
             name=exp_name,
             stop={
                 "training_iteration": 1
             },
             checkpoint_freq=1,
-            local_dir=local_dir,
+            local_dir=absolute_local_dir,
```

Comment:
I think the point of some of these tests was to specifically test if you could pass in a relative local dir in addition to an absolute one...